.dockerignore

```diff
@@ -1,6 +1,13 @@
-target
-Dockerfile
-.dockerignore
-.git
-.github
-.gitignore
+# Exclude everything:
```

Comment:
```suggestion
# Before the docker CLI sends the context to the docker daemon, it looks for a file 
# named .dockerignore in the root directory of the context. If this file exists, the 
# CLI modifies the context to exclude files and directories that match patterns in it. 
#
# You may want to specify which files to include in the context, rather than which 
# to exclude. To achieve this, specify * as the first pattern, followed by one or 
# more ! exception patterns.
# 
# https://docs.docker.com/engine/reference/builder/#dockerignore-file

# Exclude everything:
```

Reply:
I'll add this in and approve